catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java

```diff
@@ -93,6 +93,14 @@ public void testVariableBricks() {
 		assertEquals("Variable has the wrong value after stage", 42.0,
 				dataContainer.getUserVariable("p2", sprite).getValue());
 
+		solo.goBack();
+		solo.goBack();
+		solo.clickOnText("p2");
+		solo.clickOnText("p1");
+		solo.clickOnText("p1");
+		solo.clickOnText(solo.getString(R.string.brick_variable_spinner_create_new_variable));
+		solo.clickOnText(solo.getString(R.string.cancel_button));
+		solo.clickOnText("p1");
 	}
```

Comment:
I think in this case asserts are not needed because when the clickOnText-Methods are succesfull the app's behavior is correct.